src/operator/controllers/access_annotation/intents.go

```diff
@@ -0,0 +1,104 @@
+package access_annotation
+
+import (
+	"context"
+	otterizev1alpha3 "github.com/otterize/intents-operator/src/operator/api/v1alpha3"
+	"github.com/otterize/intents-operator/src/shared/errors"
+	"github.com/otterize/intents-operator/src/shared/injectablerecorder"
+	"github.com/otterize/intents-operator/src/shared/serviceidresolver/serviceidentity"
+	"github.com/samber/lo"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Intent struct {
```

Comment:
Please consider changing this name because we already have several "intent" structs :-)

Reply:
Done